### DISCUSSION TITLE
Tabs - allow constant child content

### DISCUSSION THREAD

**Discussion by Ojay** [heroui-inc/heroui#2569] [Feature requests]
I have a design where I need to display some ever-present content between the tabs and the tab panels. This content should appear regardless of what tab is selected. So something like the following...

```
      <Tabs
        variant='underlined'
      >
        <div>CONTENT TO REMAIN VISIBLE</div>
        {elements.map((one, index) => (
          <Tab key={one.key} title={one.key}>
            {one.props.children}
          </Tab>
        ))}
      </Tabs>
```

The above returns the error: `Error: Unknown element <div> in collection.`

Here's an example of achieving this with HeadlessUI...

```
<Tab.Group defaultIndex={initialTab}>
  <Tab.List
    as="nav"
  >
    {children.map((tab) => {
      const { label } = tab.props;
      const { title } = tab.props;
      return (
        <Tab
          key={label}
        >
          {title}
        </Tab>
      );
    })}
  </Tab.List>
  <div>CONTENT TO REMAIN VISIBLE</div>
  <Tab.Panels>
    {children.map((one) => (
      <Tab.Panel key={one.props.label}>{one.props.children}</Tab.Panel>
    ))}
  </Tab.Panels>
</Tab.Group>;

```

**Comment by J4v4Scr1pt** [comment]
Is this what you need? --> https://github.com/nextui-org/nextui/discussions/2493

Edit: Maybe not, think I misunderstood your question 😅

**Reply by Ojay** [reply]
Thanks for getting back to me but yeah, a little misunderstanding I think. All good 👍
